odbc/src/api/data.rs

```diff
@@ -154,18 +154,19 @@ pub unsafe fn format_and_return_bson(
         }
         _ => {}
     }
+    use i32_len::*;
     match target_type {
         CDataType::Char | CDataType::Binary => set_output_string(
             &data.to_string(),
             target_value_ptr as *mut _,
             buffer_len as usize,
-            str_len_or_ind_ptr as *mut _,
```

Comment:
We should generally be skeptical of casts like this. I had been skeptical and thought it worked, but something was wrong with the test I did, somehow.